app/src/test/java/uk/ac/ebi/atlas/search/geneids/GeneIdSearchDaoIT.java

```diff
@@ -102,7 +103,7 @@ void ifNoDocumentsAreFoundInTheIntersectionReturnEmptySet() {
                             .findAny()
                             .map(tuples -> tuples.getString(BIOENTITY_IDENTIFIER.name()))
                             .orElseThrow(RuntimeException::new);
-
+            System.out.println("[[[ Searching for rogue gene ID " + geneId + " ]]]");
```

Comment:
We probably don't need this professional debugging line, right? 😜

Reply:
Right!